pkg/resourcehandlers/git/repository.go

```diff
@@ -96,6 +96,7 @@ func (r *Repository) prepare(ctx context.Context, version string) error {
 
 	if err := w.Checkout(&gogit.CheckoutOptions{
 		Branch: checkoutDestination,
+		Force:  true,
```

Comment:
Will this affect the local setup, where unstaged changes are expected?

Reply:
It appears resourceMappings from the config file are applied only in ResolveDocumentation function after the branch checkout in the cache. It is not expected that the local setup files are in a repository of the cache, where the hard reset happens